plugins/unbound/unbound_

```diff
@@ -0,0 +1,535 @@
+		done
+		p_config "num.answer.secure" "answer secure"
+		p_config "num.answer.bogus" "answer bogus"
+		p_config "num.rrset.bogus" "num rrsets marked bogus"
+		echo "graph_info answers sorted by return value. rrsets bogus is the number of rrsets marked bogus per second by the validator"
+		;;
+	by_flags)
+		echo "graph_title Unbound DNS incoming queries by flags"
+		echo "graph_args --base 1000 -l 0"
+		echo "graph_vlabel queries / second"
+		echo "graph_category dns"
+		p_config "num.query.flags.QR" "QR (query reply) flag"
+		p_config "num.query.flags.AA" "AA (auth answer) flag"
+		p_config "num.query.flags.TC" "TC (truncated) flag"
+		p_config "num.query.flags.RD" "RD (recursion desired) flag"
+		p_config "num.query.flags.RA" "RA (rec avail) flag"
+		p_config "num.query.flags.Z" "Z (zero) flag"
+		p_config "num.query.flags.AD" "AD (auth data) flag"
+		p_config "num.query.flags.CD" "CD (check disabled) flag"
+		p_config "num.query.edns.present" "EDNS OPT present"
+		p_config "num.query.edns.DO" "DO (DNSSEC OK) flag"
+		echo "graph_info This graphs plots the flags inside incoming queries. For example, if QR, AA, TC, RA, Z flags are set, the query can be rejected. RD, AD, CD and DO are legitimately set by some software."
+		;;
+	histogram)
+		echo "graph_title Unbound DNS histogram of reply time"
+		echo "graph_args --base 1000 -l 0"
+		echo "graph_vlabel queries / second"
+		echo "graph_category dns"
+		echo hcache.label "cache hits"
+		echo hcache.min 0
+		echo hcache.draw AREA
+		echo hcache.colour 999999
+		echo h64ms.label "0 msec - 66 msec"
+		echo h64ms.min 0
+		echo h64ms.draw STACK
+		echo h64ms.colour 0000FF
+		echo h128ms.label "66 msec - 131 msec"
+		echo h128ms.min 0
+		echo h128ms.colour 1F00DF
+		echo h128ms.draw STACK
+		echo h256ms.label "131 msec - 262 msec"
+		echo h256ms.min 0
+		echo h256ms.draw STACK
+		echo h256ms.colour 3F00BF
+		echo h512ms.label "262 msec - 524 msec"
+		echo h512ms.min 0
+		echo h512ms.draw STACK
+		echo h512ms.colour 5F009F
+		echo h1s.label "524 msec - 1 sec"
+		echo h1s.min 0
+		echo h1s.draw STACK
+		echo h1s.colour 7F007F
+		echo h2s.label "1 sec - 2 sec"
+		echo h2s.min 0
+		echo h2s.draw STACK
+		echo h2s.colour 9F005F
+		echo h4s.label "2 sec - 4 sec"
+		echo h4s.min 0
+		echo h4s.draw STACK
+		echo h4s.colour BF003F
+		echo h8s.label "4 sec - 8 sec"
+		echo h8s.min 0
+		echo h8s.draw STACK
+		echo h8s.colour DF001F
+		echo h16s.label "8 sec - ..."
+		echo h16s.min 0
+		echo h16s.draw STACK
+		echo h16s.colour FF0000
+		echo "graph_info Histogram of the reply times for queries."
+		;;
+	esac
+
+	exit 0
+fi
+
+# do the stats itself
+get_state
+
+# get the time elapsed
+get_value "time.elapsed"
+if test $value = 0 || test $value = "0.000000"; then
+	echo "error: time elapsed 0 or could not retrieve data"
+	exit 1
+fi
+elapsed="$value"
+
+# print value for $1 / elapsed
+print_qps ( ) {
+	mn=`echo $1 | sed $ABBREV | tr . _`
+	get_value $1
+	echo "$mn.value" `echo scale=6';' $value / $elapsed | bc `
```

Comment:
Maybe use `awk` instead: it is expected to be available everywhere. For `bc` you would need another `autoconf` test.

Here it could be: `awk '{printf("%.6f\n", $1 / $2)}'`